src/main/java/org/amahi/anywhere/fragment/AlertDialogFragment.java

```diff
@@ -29,7 +29,7 @@
     public static final int DUPLICATE_FILE_DIALOG = 1;
     public static final int SIGN_OUT_DIALOG = 3;
     public static final int FILE_INFO_DIALOG = 2;
-    public static final String LAST_OPENED_NULL = "Never opened";
+    public final String LAST_OPENED_NULL = getString(R.string.never_opened);
```

Comment:
I would recommend that you must remove the reference to `LAST_OPENED_NULL` variable and rather use `getString` directly.

Reply:
@chirag-jn 
Done sorry for the delay.